### PR TITLE
change paths

### PR DESCRIPTION
**Pull request by jspaine**

### CODE REVIEW

**Comment by jspaine** [comment]
It seems fine like this for me, are you running both npm run build and npm start with NODE_ENV=production?
(It'll probably not work if you try after merging the heroku PR either, I renamed /dist/assets to /dist/client, so fix that in express.js)

**Comment by kenjiO** [comment]
I had to make the changes in 7da64bf to get this to work when merging this into the latest staging branch (3cfa1b3).  Can you try merging this into your staging and seeing if it works for you too?

File: server/config/express.js [hunk 1]
```diff
@@ -95,8 +95,8 @@ export default function() {
   app.use('/api', apiRoutes(API_DELAY, API_FAILURE_RATE))
 
   // Setting the static folder
-  const static_assets_path = (process.env.NODE_ENV === 'production') ? './dist/client' : './assets'
-  app.use(express.static(path.resolve(static_assets_path)))
+  if (process.env.NODE_ENV === 'production')
```
Comment (jspaine) [inline server/config/express.js]: Only need to serve static files in production, webpack dev server does it itself

File: server/config/express.js [hunk 2]
```diff
@@ -114,11 +114,6 @@ export default function() {
     })
   })
 
-  // let the client handle it
```
Comment (kenjiO) [inline server/config/express.js]: Don't we still need this? When I take this out it breaks going directly to URLs for the application pages such as when I paste http://localhost:3000/users/signin into my browser I get not found.

File: server/config/express.js [hunk 3]
```diff
@@ -113,11 +114,6 @@ export default function() {
...
-  app.use(function(req, res) {
-    res.sendFile(path.resolve('./client/index.html'))
-  })
```
Comment (kenjiO) [inline server/config/express.js:119]: Don't we still need this? When I take this out it breaks going directly to URLs for the application pages such as when I paste http://localhost:3000/users/signin into my browser I get not found.

Comment (jspaine) [reply server/config/express.js:119]: Ah, I added it back in, I think this'll do it!

**Pull request merged**